src/Layer.js

```diff
@@ -558,42 +558,42 @@ export default class Layer {
 
     _checkId (id) {
         if (id === undefined) {
-            throw new CartoValidationError('\'id\'', cvt.MISSING_REQUIRED);
+            throw new CartoValidationError('\'id\'', CartoValidationTypes.MISSING_REQUIRED);
```

Comment:
Hi @elenatorro I see the value in separating the error type in a new field, apart from the message. That's a good addition.

But I'm not sure about 2 things:
- error type names (and this was there before this PR)
- not using an alias

Regarding the first, I think we could use the format *BaseNameErrorTypes* (so `CartoValidationErrorTypes` instead of `CartoValidationTypes, `CartoMapsAPIErrorTypes` instead of just `CartoMapsAPITypes`... and so on).

About the alias, don't you think it is more practical to keep them? using `throw new CartoValidationError('\'id\'', cve.MISSING_REQUIRED);` instead of `throw new CartoValidationError('\'id\'', CartoValidationTypes.MISSING_REQUIRED);` I see this as quite similar to the 'exp' alias we use extensively.

Reply:
I thought the same about adding the `Error` in the constant name, I totally agree with that!

About the alias, I did it to improve readability because `cvt` or `exp` don't mean anything, having long meaningful names is always better than having abbreviations:

https://readlearncode.com/code-and-stuff/clean-code-variables/

> Use words that can be pronounced and avoid abbreviations
There are many common abbreviations that are well known by everyone, but many that are not. Avoid using any abbreviations, for example, its better to use clientDateOfBirth rather that cDOB. Imagine that you must talk about your code in a meeting with people who are not familiar with what your code does. It’s much clearer to say “clientDateOfBirth” than “cDOB” every time you want to refer to that variable.